hack/upgrade-test-harness/k8s/kubectl.go

```diff
@@ -117,7 +117,12 @@ func (h *Kubectl) GetPods(namespace string, labelSelector string) ([]corev1.Pod,
 
 // LoadResources loads manifests from the given file path.
 func (h *Kubectl) LoadResources(filePath string) (*resource.Result, error) {
-	validator, err := h.factory.Validator(true)
+	dynamicClient, err := h.factory.DynamicClient()
+	if err != nil {
+		return nil, err
+	}
+	verifier := resource.NewQueryParamVerifier(dynamicClient, h.factory.OpenAPIGetter(), resource.QueryParamFieldValidation)
+	validator, err := h.factory.Validator(metav1.FieldValidationStrict, verifier)
```

Comment:
Did you forget to commit `go.mod` because this PR still has
```go.mod
require (
	k8s.io/api v0.22.4
	k8s.io/apimachinery v0.22.4
	k8s.io/cli-runtime v0.22.4
	k8s.io/client-go v0.22.4
	k8s.io/kubectl v0.22.4
)
```